Plan/fabric/src/main/java/net/playeranalytics/plan/commands/CommandManager.java

```diff
@@ -77,7 +81,11 @@ private int execute(CommandContext<ServerCommandSource> ctx, Subcommand subcomma
             try {
                 subcommand.getExecutor().accept((CMDSender) ctx.getSource(), new Arguments(getCommandArguments(ctx)));
             } catch (Exception e) {
-                ctx.getSource().sendError(new LiteralText(e.getMessage()));
+                ctx.getSource().sendError(new LiteralText("An internal error occurred, see the console for details."));
```

Comment:
I left this in because Fabric _probably_ doesn't print any "an internal error occurred while attempting to perform this command" stuff 😅